docs/UserGuide.md

```diff
@@ -206,6 +206,21 @@ Examples:
 * `markOrder 1` marks the order with index 1 from the displayed order list as completed.
 * `listOrder 1` followed by `markOrder 2` selects the 1st person in the address book and marks the 2nd order under the 1st person as completed.
 
+### Reverts an order to pending status : `unmarkOrder`
+
+Reverts an order under the person whose orders are displayed as pending.
+
+Format: `unmarkOrder ORDER_INDEX`
+
+* A person's order list must be selected using `listOrder` before marking an order from that person.
```

Comment:
![image](https://github.com/user-attachments/assets/48eb16aa-d4b7-4280-88be-63695fb85b41)
here is some phrasing that kx changed which i think makes sense with his new command

Reply:
do change to standardise